mavros_msgs/msg/Obstacle3D.msg

```diff
@@ -0,0 +1,25 @@
+# Obstacle Distance 3D message
+# Represents a single obstacle position in 3D space
+# 
+# https://mavlink.io/en/messages/ardupilotmega.html#OBSTACLE_DISTANCE_3D
+# MAVLink Message ID: 11037
+
+std_msgs/Header header
+
+# MAV_DISTANCE_SENSOR enum
+uint8 sensor_type
+
+# MAV_FRAME enum  
+uint8 frame
+
+# Unique ID for the obstacle
+uint16 obstacle_id
+
+# Obstacle position in meters
+float32 x
+float32 y
+float32 z
```

Comment:
Better to use Point or Vector3.

https://docs.ros2.org/foxy/api/geometry_msgs/msg/Point.html